pysonde/_proc_level2.py

```diff
@@ -387,6 +387,16 @@ def convert_num2_date_with_nan(num, format):
     # merged_conf._set_parent(OmegaConf.merge(config, meta_data_cfg))
     ds_interp.attrs = ds.attrs
 
+    # Replace placeholders in global attributes
+    for key, value in ds_interp.attrs.items():
+        if isinstance(value, str):  # Only process string attributes
```

Comment:
The code is self descriptive
```suggestion
        if isinstance(value, str):
```